concept/error.rs

```diff
@@ -107,6 +112,10 @@ pub enum ConceptWriteError {
 
     SetHasOrderedOwnsUnordered {},
     SetHasUnorderedOwnsOrdered {},
+
+    Conversion {
```

Comment:
I return this error from `Into<Annotation>` conversions of `PlaysAnnotation`, etc. Open for better naming. Thought about just `Internal` stuff, but `ConceptWriteError` is already `Internal` enough.